l10n_co_geo/data/res_country_state.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see they have an ambassador for Colombia already, the problem is I guess, that ZIP codes are not openly licensed.